netbench-orchestrator/src/orchestrator.rs

```diff
@@ -7,10 +7,17 @@ mod error;
 mod report;
 mod state;
 
+use crate::{
+    ec2_utils, ec2_utils::InfraDetail, s3_utils, ssm_utils, ssm_utils::NetbenchDriverType,
+};
+use aws_sdk_s3::primitives::ByteStream;
+use tracing::info;
+
 pub use cli::{Cli, HostConfig, OrchestratorConfig};
 pub use error::{OrchError, OrchResult};
 pub use state::STATE;
 
+#[allow(dead_code)]
```

Comment:
how would this get used?

Reply:
Its used [here](https://github.com/aws/s2n-netbench/blob/bb76621b1a0be9b58262a3da483d720972863231/netbench-orchestrator/src/main.rs#L42). clippy complains about the unused variant `TestInfra` in this PR since we are using the `Full` mode.

Reply:
I mean how would switching between RunModes be done?

Reply:
Oh its currently only useful when developing. Its not exposed to an actual user of the orchestrator (the user is not testing and doesn't need to just launch/cleanup infra.. not yet)

Reply:
can you put the `#[allow(dead_code)]` on the Enum variant then, and add a comment suggesting where to set the mode when developing (in main.rs)